snap/snapcraft.yaml

```diff
@@ -39,6 +39,7 @@ parts:
         -DLLVM_ROOT_DIR=../../llvm/install \
         -DBUILD_LTO_LIBS=ON \
         -DLDC_INSTALL_LTOPLUGIN=ON \
+        -DLDC_INSTALL_LLVM_RUNTIME_LIBS=ON \
```

Comment:
This makes sure the LLVM compiler-rt libraries are included; they are needed for PGO, sanitizers etc. These libs are affected by https://github.com/ldc-developers/ldc/issues/3054 though as LLVM currently isn't compiled with `-Wa,-mrelax-relocations=no`.

Reply:
Is that something we can address on the package side?  Bearing in mind that we build LLVM in the package?

Reply:
Sure, we'd simply need that extra C[++] flag when building LLVM too. I just don't want to override the default flags and am unsure as to whether setting `CMAKE_C_FLAGS` would override them.